pkg/config/config.go

```diff
@@ -0,0 +1,53 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+)
+
+// Config is global config object
+var Config *config
+
+type config struct {
+	FeatureGates map[string]bool `json:"featureGates,omitempty"`
+}
+
+func init() {
+	Config = newConfig()
+}
+
+func newConfig() *config {
+	if Config != nil {
+		return Config
+	}
+	newCfg := &config{}
+	newCfg.FeatureGates = make(map[string]bool)
+	return newCfg
+}
+
+// ReadConfig loads config
+func (cfg *config) ReadConfig(configFile string) error {
```

Comment:
There is a `readConfig` in manager.go file to parse the `resourceList` data in configMap.
I think perhaps we want to combine both `resourceList` and `featureGates` to the same `readConfig` function and make it a sub-element of `resourceManager`.

Reply:
What I intended, was to make this config as independent of the existing 'manager config' as possible as it is still yet to be decided if if would like to use one big global config, or for example separate the device-discovery-related config and feature-related config (which I think is optional by it's own nature).